ReadMe.md

```diff
@@ -55,69 +55,69 @@ For CLI usage please see [Manual.md](Manual.md).
+                                 # You shall run `clickhouse-backup delete local <backup_name>` command to remove temporary backup files from the local disk
+  backups_to_keep_remote: 0      # BACKUPS_TO_KEEP_REMOTE, how many latest backup should be kept on remote storage, 0 means all uploaded backups will be stored on remote storage. 
+                                 # If old backups are required for newer incremental backup then it won't be deleted. Be careful with long incremental backup sequences.
+  log_level: info                # LOG_LEVEL, a choice from `debug`, `info`, `warn`, `error`
   allow_empty_backups: false     # ALLOW_EMPTY_BACKUPS
   # concurrency means parallel tables and parallel parts inside tables
-  # for example 4 means max 4 parallel tables and 4 parallel parts inside one table, so equal 16 streams
-  download_concurrency: 1        # DOWNLOAD_CONCURRENCY, max 255, default value round(sqrt(AVAILABLE_CPU_CORES / 2))  
-  upload_concurrency: 1          # UPLOAD_CONCURRENCY, max 255, default value round(sqrt(AVAILABLE_CPU_CORES / 2))
+  # for example 4 means max 4 parallel tables and 4 parallel parts inside one table, so equals 16 concurrent streams
+  download_concurrency: 1        # DOWNLOAD_CONCURRENCY, max 255, by default, the value is round(sqrt(AVAILABLE_CPU_CORES / 2))  
+  upload_concurrency: 1          # UPLOAD_CONCURRENCY, max 255, by default, the value is round(sqrt(AVAILABLE_CPU_CORES / 2))
 
-  # RESTORE_SCHEMA_ON_CLUSTER, execute all schema related SQL queries with `ON CLUSTER` clause as Distributed DDL, 
-  # look to `system.clusters` table for proper cluster name, also system.macros could be used, 
-  # don't applicable when `use_embedded_backup_restore: true`
+  # RESTORE_SCHEMA_ON_CLUSTER, execute all schema related SQL queries with `ON CLUSTER` clause as Distributed DDL. 
+  # Check `system.clusters` table for the correct cluster name, also `system.macros` can be used.
+  # This isn't applicable when `use_embedded_backup_restore: true`
   restore_schema_on_cluster: ""   
   upload_by_part: true           # UPLOAD_BY_PART
   download_by_part: true         # DOWNLOAD_BY_PART
-  use_resumable_state: true      # USE_RESUMABLE_STATE, allow resume upload and download according to <backup_name>.resumable file
+  use_resumable_state: true      # USE_RESUMABLE_STATE, allow resume upload and download according to the <backup_name>.resumable file
 
-  # RESTORE_DATABASE_MAPPING, restore rules from backup databases to target databases, which is useful on change destination database all atomic tables will create with new uuid.
-  # for environment use following format "src_db1:target_db1,src_db2:target_db2" 
+  # RESTORE_DATABASE_MAPPING, restore rules from backup databases to target databases, which is useful when changing destination database, all atomic tables will be created with new UUIDs.
+  # Use the following format "src_db1:target_db1,src_db2:target_db2" 
```

Comment:
no, `for environment` here means for environment variable RESTORE_DATABASE_MAPPING

for config file you shall continue use map in YAML

Reply:
But it is self-evident, this comment is specifically for the RESTORE_DATABASE_MAPPING. `for environment` usually mean things like `prod`, `dev` etc. 

If you like to keep this, I suggest: `for environment -> for this environment variable`

Reply:
Yes, agree, we need to change text 
`when use environment variabl, use following format`